dev/devicelab/bin/tasks/flutter_test_performance.dart

```diff
@@ -37,68 +37,26 @@ enum TestStep {
 }
 
 Future<int> runTest({bool coverage = false}) async {
-  final Stopwatch clock = Stopwatch()..start();
   final List<String> arguments = <String>[
     'test',
+    '--benchmark'
   ];
   if (coverage) {
     arguments.add('--coverage');
   }
+  final String workingDirectory = path.join(flutterDirectory.path, 'dev', 'automated_tests');
   arguments.add(path.join('flutter_test', 'trivial_widget_test.dart'));
   final Process analysis = await startProcess(
     path.join(flutterDirectory.path, 'bin', 'flutter'),
     arguments,
-    workingDirectory: path.join(flutterDirectory.path, 'dev', 'automated_tests'),
+    workingDirectory: workingDirectory
   );
-  int badLines = 0;
-  TestStep step = TestStep.starting;
-  await for (String entry in analysis.stdout.transform<String>(utf8.decoder).transform<String>(const LineSplitter())) {
```

Comment:
Discussed this with @Hixie -- this test of the output was here intentionally to keep us from missing the case of performance numbers looking artificially ok due to some other factor that would be visible in the test output.  So we should keep this logic testing in here.

I thought we only downloaded the web sdk when you need to build, not proactively if you're on a host that has Chrome?

Reply:
> I thought we only downloaded the web sdk when you need to build, not proactively if you're on a host that has Chrome?

I checked and a few weeks back I made it always download for tests. I can tweak that instead, but I think that this change didn't cause the test to fail reliably because that output will only appear if another test has not downloaded it already. Basically the regexp approach is just bad

> this test of the output was here intentionally to keep us from missing the case of performance numbers looking artificially ok due to some other factor that would be visible in the test output. So we should keep this logic testing in here

What?